packages/tasks/src/tasks/document-question-answering/inference.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Inference code generated from the JSON schema spec in ./spec
+ *
+ * Using src/scripts/inference-codegen
+ */
+
+/**
+ * Inputs for Document Question Answering inference
+ */
+export interface DocumentQuestionAnsweringInput {
+	/**
+	 * One or several document+question pairs to answer
+	 */
+	inputs: DocumentQuestionAnsweringInpu[] | DocumentQuestionAnsweringInpu;
```

Comment:
`DocumentQuestionAnsweringInpu` truncating the name to declare the second class is done by quicktype I guess. Would it be possible to rename it to `DocumentQuestionAnsweringInputItem` for example? (by adding an explicit `title` field)

Reply:
Handled in 6558af4 - good catch!

Reply:
According to this doc: https://huggingface.co/docs/transformers/v4.37.1/en/main_classes/pipelines#transformers.DepthEstimationPipeline

The output is an image, so I specced it with the same type as image inputs